code/modules/client/preferences_gear.dm

```diff
@@ -449,6 +449,12 @@ var/global/list/gear_datums = list()
 	slot = slot_w_uniform
 	cost = 1
 
+/datum/gear/Kimono
```

Comment:
Uppercase in path names is not really standard, no idea if Urist cares about that though.